mm2src/common/common.rs

```diff
@@ -375,6 +375,7 @@ pub fn stack_trace_frame(instr_ptr: *mut c_void, buf: &mut dyn Write, symbol: &b
     // Skip common and less than informative frames.
 
     match name {
+        // FIXME: What mm2 here refers to?
         "mm2::crash_reports::rust_seh_handler"
```

Comment:
is mm2 here the `[bin]` (maybe should be `kdf` then?) or the path `mm2::` removed in this PR?

Reply:
it seems that `crash_reports` was a file in `mm2_main` but moved to `common` crate, we should fix it here so that `rust_seh_handler` is skipped in the stack trace. Same for `signal_handler` https://github.com/KomodoPlatform/komodo-defi-framework/blob/527642e053052eae68a03de002d9deff3c10bf8f/mm2src/common/common.rs#L400-L401
Please also merge with latest dev to make sure no problems arise and to check the tests in CI. I will approve and merge the PR after that.